apps/full-site-editing/full-site-editing-plugin/lib/fetch-post.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable wpcalypso/import-docblock */
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { select } from '@wordpress/data';
+
+export const fetchPost = async ( postId, postType ) => {
+	const postTypeObject =
+		select( 'core' ).getEntityRecord( 'root', 'postType', postType ) ||
+		( await apiFetch( {
+			path: `/wp/v2/types/${ postType }`,
+		} ) );
+
+	if ( ! postTypeObject ) {
+		return null;
+	}
+
+	return await apiFetch( {
+		path: `/wp/v2/${ postTypeObject.rest_base }/${ postId }`,
+	} );
+};
```

Comment:
can we explain why this exists and why we aren't using `getEntityRecord()` to fetch the post?

Reply:
I was under the impression that `getEntityRecord` was a simple selector, but in fact it has a resolver that `apiFetch`es on its own.
I'll see to update this behaviour ASAP.